### PR TITLE
[ADF-4106] Test Role Consumer permissions

### PR DESCRIPTION
**Pull request by gmandakini**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
https://issues.alfresco.com/jira/browse/ADF-4106

### CODE REVIEW

File: e2e/content-services/permissions-component.e2e.ts [hunk 1]
```diff
@@ -245,4 +261,56 @@ describe('Permissions Component', function () {
...
+            permissionsPage.clickRoleDropdown();
+            permissionsPage.selectOption('Consumer');
+            done();
+        });
+
+        it('[C274691] Should not to be able to Delete, see Info or Upload a file with Consumer permission', () => {
+
+            loginPage.loginToContentServicesUsingUserModel(filePermissionUser);
+            contentServicesPage.goToDocumentList();
+            browser.sleep(10000);
```
Comment (adinapitul) [inline e2e/content-services/permissions-component.e2e.ts]: can you remove this sleep and instead wait for a specific element? maybe waitForTableBody ?

Comment (gmandakini) [reply e2e/content-services/permissions-component.e2e.ts]: I couldn't find another workaround to do this, as it's dependent on the search indexing to finish beneath, I think.

Comment (adinapitul) [reply e2e/content-services/permissions-component.e2e.ts]: For indexing to finish you have a sleep above, at line 89.

File: e2e/content-services/permissions-component.e2e.ts [hunk 2]
```diff
@@ -245,4 +261,56 @@ describe('Permissions Component', function () {
...
+            permissionsPage.clickRoleDropdown();
+            permissionsPage.selectOption('Consumer');
+            done();
+        });
+
+        it('[C274691] Should not to be able to Delete, see Info or Upload a file with Consumer permission', () => {
+
+            loginPage.loginToContentServicesUsingUserModel(filePermissionUser);
+            contentServicesPage.goToDocumentList();
+            browser.sleep(15000); // for the Search indexing to finish
```
Comment (cristinaj) [inline e2e/content-services/permissions-component.e2e.ts]: This sleep is not needed, the indexing is done only after creating the folder.

File: e2e/content-services/permissions-component.e2e.ts [hunk 3]
```diff
@@ -245,4 +261,56 @@ describe('Permissions Component', function () {
...
+            permissionsPage.checkSearchUserInputIsDisplayed();
+            permissionsPage.searchUserOrGroup(filePermissionUser.getId());
+            permissionsPage.clickUserOrGroup(filePermissionUser.getFirstName());
+            permissionsPage.checkUserOrGroupIsAdded(filePermissionUser.getId());
+            permissionsPage.clickRoleDropdown();
+            permissionsPage.selectOption('Consumer');
+            done();
+        });
+
+        it('[C274691] Should not to be able to Delete, see Info or Upload a file with Consumer permission', () => {
```
Comment (cristinaj) [inline e2e/content-services/permissions-component.e2e.ts]: The testrail id is not correct.

File: e2e/content-services/permissions-component.e2e.ts [hunk 4]
```diff
@@ -245,4 +261,56 @@ describe('Permissions Component', function () {
...
+                .enterText(roleFolderModel.name)
+                .resultTableContainsRow(roleFolderModel.name)
+                .clickOnSpecificRow(roleFolderModel.name);
+            contentList.checkContentIsDisplayed('Role' + fileModel.name);
+            contentList.doubleClickRow('Role' + fileModel.name);
+            viewerPage.checkFileIsLoaded();
+            viewerPage.clickCloseButton();
+            contentList.waitForTableBody();
+            contentList.checkDeleteIsDisabled('Role' + fileModel.name);
+            browser.refresh();
```
Comment (cristinaj) [inline e2e/content-services/permissions-component.e2e.ts]: Is this refresh() needed?

File: e2e/content-services/permissions-component.e2e.ts (hunk 2 above)
Comment (gmandakini) [reply e2e/content-services/permissions-component.e2e.ts]: Please let me know, if there is a workaround for this.

File: e2e/content-services/permissions-component.e2e.ts (hunk 3 above)
Comment (gmandakini) [reply e2e/content-services/permissions-component.e2e.ts]: updated. thanks

File: e2e/content-services/permissions-component.e2e.ts (hunk 4 above)
Comment (gmandakini) [reply e2e/content-services/permissions-component.e2e.ts]: I have removed browser refresh and added the Protractor Escape key and a check to make sure, the action menu is not displayed anymore, before carrying on furtheractions.

File: e2e/content-services/permissions-component.e2e.ts (hunk 1 above)
Comment (gmandakini) [reply e2e/content-services/permissions-component.e2e.ts]: yes, tried using just that sleep, but the test fails without this sleep.

File: e2e/content-services/permissions-component.e2e.ts (hunk 2 above)
Comment (cristinaj) [reply e2e/content-services/permissions-component.e2e.ts]: I couldn't find one, it needs more investigation. I'm fine keeping it for now.

**Pull request closed without merging**